models/brac/views/hhview_visits.sql

```diff
@@ -0,0 +1,29 @@
+SELECT reported, reported_by_parent, household_id FROM(
+
+SELECT 
+	upr.reported AS reported,
+	upr.reported_by_parent,
+	cvp.parent_uuid AS household_id
+FROM 
+	{{ ref("useview_patient_record") }} AS upr
+INNER JOIN {{ ref("contactview_person") }} cvp ON (upr.patient_id=cvp.patient_id)
```

Comment:
indenation
explicitly call out the alias i.e AS cvp
braces around the join condition are unnecessary